cloud: categorize FAQs (#10218)
pingcap/docs: tidb-cloud/tidb-cloud-faq.md

TiFlash is the analytical storage that replicates data from the transactional store (TiKV) in real-time and supports real-time OLAP workloads. Unlike TiKV, TiFlash stores data in columns to accelerate analytical processing. TiFlash is also horizontally scalable; increasing TiFlash nodes will increase OLAP storage and computing capacity.

### How does TiDB replicate data between the TiKV nodes?

TiKV divides the key-value space into key ranges, and each key range is treated as a “Region”. In TiKV, data is distributed among all nodes in a cluster and uses the Region as the basic unit. PD is responsible for spreading (scheduling) Regions as evenly as possible across all nodes in a cluster.

TiDB uses the Raft consensus algorithm to replicate data by Regions. Multiple replicas of a Region stored in different nodes form a Raft Group.

Each data change is recorded as a Raft log. Through Raft log replication, data is safely and reliably replicated to multiple nodes of the Raft Group.

> ## High Availability FAQ

Author: ```suggestion
## High availability FAQ
```

> ## High Availability FAQs

Reviewer: ```suggestion
## High Availability FAQ
```

### How does TiDB Cloud ensure high availability?

TiDB uses the Raft consensus algorithm to ensure that data is highly available and safely replicated throughout storage in Raft Groups. Data is redundantly copied between TiKV nodes and placed in different Availability Zones to protect against machine or data center failure. With automatic failover, TiDB ensures that your service is always on.

> As a Software as a Service (SaaS) provider, we take data security seriously. We have established strict information security policies and procedures required by the [Service Organization Control (SOC) 2 Type 1 compliance](https://pingcap.com/blog/pingcap-successfully-completes-soc-2-type-1-examination-for-tidb-cloud). This ensures that your data is secure, available, and confidential.

Reviewer: ```suggestion
As a Software as a Service (SaaS) provider, we take data security seriously. We have established strict information security policies and procedures required by the [Service Organization Control (SOC) 2 Type 1 compliance](https://en.pingcap.com/press-release/pingcap-successfully-completes-soc-2-type-1-examination-for-tidb-cloud/). This ensures that your data is secure, available, and confidential.
```

## Migration FAQs

### Is there an easy migration path from another RDBMS to TiDB Cloud?

TiDB is highly compatible with MySQL. You can migrate data from any MySQL-compatible databases to TiDB smoothly, whether the data is from a self-hosted MySQL instance or RDS service provided by the public cloud. For more information, see [Migrate Data from MySQL-Compatible Databases](/tidb-cloud/migrate-data-into-tidb.md).

> ## Feature FAQs

Reviewer: This is a bit confusing to me.

> ## Feature FAQs

Author: ```suggestion
## HTAP FAQs
```

> ### How do I make use of TiDB Cloud’s HTAP capabilities?

Reviewer: ```suggestion
### How do I make use of TiDB Cloud's HTAP capabilities?
```

### Can I import my data directly to TiFlash?

No. When you import data to TiDB Cloud, the data is imported to TiKV. After the import is complete, you can use SQL statements to specify which tables to be replicated to TiFlash. Then, TiDB will create the replicas of the specified tables in TiFlash accordingly. For more information, see [Create TiFlash Replicas](/tiflash/create-tiflash-replicas.md).

### Can I export TiFlash data in the CSV format?

No. TiFlash data cannot be exported.

## Security FAQs

### How does TiDB protect data privacy and ensure security?

Transport Layer Security (TLS) and Transparent Data Encryption (TDE) are included for encryption at rest. There are two different network planes: the application to the TiDB server and the plane for data communication. We include extended syntax to compare Subject Alternative Name for verification of certificates and TLS context for internal communication.

### Can TiDB Cloud run in our VPC?

No. TiDB Cloud runs on the PingCAP VPC, but the data and traffic are encrypted by default. So you do not need to worry about data privacy issues.

> ## Support FAQs

Reviewer: ```suggestion
## Support FAQ
```

### What support is available for customers?

> TiDB Cloud is supported by the same team behind TiDB, which has run mission-critical use cases for over 1500 global enterprises across industries including financial services, e-commerce, enterprise applications, and gaming. Support is available 24/7 for TiDB Cloud users.

Reviewer 2: ```suggestion
TiDB Cloud is supported by the same team behind TiDB, which has run mission-critical use cases for over 1500 global enterprises across industries including financial services, e-commerce, enterprise applications, and gaming. Support is available 24/7 for TiDB Cloud customers.
```
